bugs.h

```diff
@@ -1,4 +1,15 @@
+#pragma once
+
 #include <map>
-#include "owner.h"
+#include <string>
+
+struct bug
+{
+	std::string package;
+	std::string bugno;
+	bug(std::string, std::string);
+};
```

Comment:
The plan was to maybe also print the package name in
the "report"

But I don't know how it will perceived, maybe like some kind
of passive aggressive attack.

The owner of a bug can be changed easily and fastly
 in the BTS, not in a compiled package.